### ISSUE TITLE
Search component should have its ARIA role in the <form> element

### ISSUE THREAD

**Issue by thisisdano** [uswds/uswds#3344]
Let's not use a separate `<div>` for our `role="search"` markup. Each search variant should be consistent and put the ARIA role in the `<form>` element.

**Comment by thisisdano** [comment]
Done in USWDS 2.6.0: https://github.com/uswds/uswds/commit/8b8d02fd31e68ff3ada5530db627680845fcc13e
